docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField

```

Comment:
### Use Input Properties in TextField

P capital for consistency sake with ### TextField Properties

Reply:
Done.